test/e2e/s3/test.js

```diff
@@ -340,6 +340,8 @@ describe('s3 support', () => {
         ))
     );
 
+    await api.apiPostJson(`projects/${projectId}/forms/${xmlFormId}/draft/publish`, {});
```

Comment:
Did changing this change something?

Reply:
Yes - it's publishing the form so that non-`/draft` endpoints can be used.

Reply:
Is it necessary?